commandapi-core/src/main/java/dev/jorel/commandapi/arguments/CommandAPIArgumentType.java

```diff
@@ -82,6 +82,11 @@ public enum CommandAPIArgumentType {
 	 */
 	CHATCOLOR("minecraft:color"),
 
+	/**
+	 * The CommandArgument
+	 */
+	COMMAND,
```

Comment:
This needs a string name and an implementation in `commandapi-plugin`'s `AdvancedConverter.java` file. The name `api:command` is sufficient. Adding this allows the plugin command converter to use the command argument.

I'm aware that `LIST` doesn't have one - that's simply because the list argument is much much more complex than this argument and I can see this argument being used (e.g. for a plugin converted `/sudo` command)